force-app/main/default/classes/MeetupTriggerHandler.cls

```diff
@@ -0,0 +1,43 @@
+/* 
+* @description Responsible for all Meetup sObject trigger behaviors.
+*/
+public with sharing class MeetupTriggerHandler {
+    public void onBeforeInsert(List<Meetup__c> meetupRecords) {
+        generateRegistrationCodes(meetupRecords);
+    }
+    
+    /* 
+	* @description Populates the given meetup records with a unique Registration Code.
+    * @param meetupRecords The list of meetup records to operate on.
+    */
+    public void generateRegistrationCodes(List<Meetup__c> meetupRecords) {
+        Schema.DescribeFieldResult registrationCodeField = Meetup__c.RegistrationCode__c.getDescribe();
+        Integer registrationCodeLength = registrationCodeField.getLength();
```

Comment:
I decided to tie the generated string length to the custom field length, to allow admins to change them in the future.